dev/io.openliberty.wsoc.internal_fat/fat/src/io/openliberty/wsoc/tests/SecureTest.java

```diff
@@ -109,6 +109,8 @@ public static void setUp() throws Exception {
         bwst.setUp();
         // tests cannot work until ssl is up
         wlp.waitForStringInLog("CWWKO0219I:.*ssl.*");
+        // tests cannot work until ltpa keys are created
+        wlp.waitForStringInLog("CWWKS4104A:.*LTPA keys created.*");
```

Comment:
Should we wait for  `CWWKS4105I: LTPA configuration is ready after x seconds` instead?

Reply:
I'd also say a copyright update to 2021 would be good here since we're making a functional change to the test.

Reply:
Yeah I think that would be good to add. Ill go ahead and make the change